pycbc/workflow/grb_utils.py

```diff
@@ -241,20 +241,33 @@ def setup_pygrb_pp_workflow(wf, pp_dir, seg_dir, segment, bank_file,
                             insp_files, inj_files, inj_insp_files, inj_tags):
     """
     Generate post-processing section of PyGRB offline workflow
+
+    Parameters
+    ----------
+    wf : The workflow object
+    pp_dir : The directory where the post-processing files will be stored
+    seg_dir : The directory where the segment files are stored
+    segment : The segment to be analyzed
+    bank_file : The full template bank file
+    insp_files : The list of inspiral files
+    inj_files : The list of injection files
+    inj_insp_files : The list of inspiral files for injections
+    inj_tags : The list of injection tags
+
+    Returns
+    -------
+    trig_files : FileList
+        The list of combined trigger files
+        [ALL_TIMES, ONSOURCE, OFFSOURCE, OFFTRIAL_1, ..., OFFTRIAL_N]
+        FileList (N can be set by the user and is 6 by default)
+    cluster_files : FileList
```

Comment:
I would go for `clustered_files`: cluster makes me think of something that could be related to the machines we run on.

Reply:
Holds for `pycbc_make_offline_grb_workflow` as well.